docs/docs/contributing/just.md

```
@@ -0,0 +1,284 @@
+:::important[Build first]
+
+Excluding the React.js example application,
+the respective projects must be built first before you can
+run them or see the latest changes you made.
+
+:::
+
+<Tabs groupId="example-apps">
+  <TabItem value="console-cpp" label="C++ console app">
+
+    ```sh title="Run it"
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)